### ISSUE TITLE
Freshly Built Openshot Prints "no notify signal" Messages In Console

### ISSUE THREAD

**Issue by AE1020** [OpenShot/openshot-qt#4498]
**Describe the bug:**
I had a relatively painless build process of `libopenshot-audio` and `libopenshot`.  Just needed libqt5svg5...[added that to the wiki](https://github.com/OpenShot/libopenshot/wiki/Linux-Build-Instructions/_compare/7633d718cd9ecf222c0c8f2fd915f934906a60e4...f1e9890663a67aa6a1e565a29f573901d1db9ccd).  *(I also used `-DCMAKE_BUILD_TYPE=Debug`  in the makefiles to get debugging information.)*

`openshot-qt` launches, but I noticed it prints out many messages to the console like:

    Property 'modal'' of object 'TimelineWebView' has no notify signal and is not constant, value updates in HTML will be broken!
    Property 'windowModality'' of object 'TimelineWebView' has no notify signal and is not constant, value updates in HTML will be broken!
    Property 'enabled'' of object 'TimelineWebView' has no notify signal and is not constant, value updates in HTML will be broken!
    ...

The messages appear in the console, but are not in the log file in the `.openshot_qt/` directory.

**Steps to reproduce the behavior:**

    ~/openshot/openshot-qt$ python3 src/launch.py

**Expected behavior:**
No such messages.

**System Details:**
- OpenShot Version: 2.6.1-dev
- Operating System / Distro: Lubuntu 20.04

**Log Files:**
Attached is capture of the full console output.
[console.log](https://github.com/OpenShot/openshot-qt/files/7384389/console.log)

**Comment by JacksonRG** [comment]
Thanks for pointing this out. Did this cause any issues in running openshot?

**Comment by AE1020** [comment]
Hello, sorry for missing this.  No, it did not cause any issues.  But when I had some trouble with undo I looked into the source and saw there was a pretty long way to go for that feature.  So I've ended up tinkering with KdenLive, as I know C++ and am comfortable with that.

It is too bad that a framework like [Apple's CoreData](https://en.wikipedia.org/wiki/Core_Data) hasn't really gotten standardized in open source.  People are often trying to add undo and redo after the fact to an already existing application.  That's bound to be problematic when it's such a cross-cutting concern...one's application should have undo and redo from the beginning!
